File: 1672-Richest_Customer_Wealth/solution.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        maxWealthSoFar = 0
        for customers in accounts:
            currentCustomerWealth = 0
            for banks in customers:
                currentCustomerWealth +=banks
            maxWealthSoFar = max(maxWealthSoFar, currentCustomerWealth)
        return maxWealthSoFar
        
        #Time Complexity = O(n x m)
        #Space Complexity = O(1)

File: 1672-Richest_Customer_Wealth/solution.cs
public class Solution {
    public int MaximumWealth(int[][] accounts) {
        int maxWealthSoFor = 0;
        foreach(int[] customers in accounts)
        {
            int currentCustomerWealth = 0;
            foreach (int bank in customers)
            {
                currentCustomerWealth += bank;
            }
            maxWealthSoFor = Math.Max(maxWealthSoFor, currentCustomerWealth);
        }
        return maxWealthSoFor;
    }//Time Complexity = O(m x n)
        //Space Complexity = O(1)
}